### PR TITLE
split some long lines in many

### PR DESCRIPTION
**Pull request by AlessandroMinoccheri**

### CODE REVIEW

File: src/Handler/StreamHandler.php [hunk 1]
```diff
@@ -343,13 +343,21 @@ private function resolveHost(RequestInterface $request, array $options)
             if ('v4' === $options['force_ip_resolve']) {
                 $records = dns_get_record($uri->getHost(), DNS_A);
                 if (!isset($records[0]['ip'])) {
-                    throw new ConnectException(sprintf("Could not resolve IPv4 address for host '%s'", $uri->getHost()), $request);
+                    throw new ConnectException(sprintf(
```
Comment (sagikazarmark) [inline src/Handler/StreamHandler.php]: This should be

```php
throw new ConnectException(
    sprintf(
        "Could not resolve IPv4 address for host '%s'",
        $uri->getHost()
    ),
    $request
);
```

File: src/Handler/StreamHandler.php [hunk 2]
```diff
@@ -343,13 +343,21 @@ private function resolveHost(RequestInterface $request, array $options)
...
+                        $uri->getHost()),
+                        $request
+                    );
                 }
                 $uri = $uri->withHost($records[0]['ip']);
             } elseif ('v6' === $options['force_ip_resolve']) {
                 $records = dns_get_record($uri->getHost(), DNS_AAAA);
                 if (!isset($records[0]['ipv6'])) {
-                    throw new ConnectException(sprintf("Could not resolve IPv6 address for host '%s'", $uri->getHost()), $request);
+                    throw new ConnectException(sprintf(
```
Comment (sagikazarmark) [inline src/Handler/StreamHandler.php]: Same as above

**Comment by AlessandroMinoccheri** [comment]
Yeah, you are right, I have corrected the lines commented!

**Comment by sagikazarmark** [comment]
Great, thanks a lot!

**Comment by wirwolf** [comment]
Hello @AlessandroMinoccheri, i create a fork of this repo and release this changes into the new version(6.4.0).

Namespace compatibility is 100%, and you can use my changes
* replace `"guzzlehttp/guzzle": "^6.3"` to `"someblackmagic/guzzle": "^6.4"` 
* add `"guzzlehttp/guzzle": "6.3.*"` into replace block in your composer.json

Fork link: https://github.com/SomeBlackMagic/guzzle/

**Pull request merged**